docs/modules/ROOT/pages/policy_input.adoc

```diff
@@ -0,0 +1,74 @@
+= Policy Input
+
+The policy input refers the variable `input` in rego. This page describes the structure of the input
+available for policy evaluation when using the Enterprise Contract CLI. The input is different for
+each subcommand as each subcommand has a different specialty.
+
+== Validate Image
+
+The input format for the `validate image` command describes all the information related to the image
+being available. Not all information is available for all images. This is noted below in the
+description of each attribute.
+
+It is important to note that this command may proccess multiple images at the same time. In such
+cases, a different input is generated for each image. In other words, policy rules only eve have
+access to the information about a single image.
+
+[,json]
```

Comment:
We could have a proper jsonschema but this might be sufficient until that need becomes obvious. What do other think of this approach?

Reply:
Works for now